File: Programmers/Lv.0/마지막 두 원소/181927.java
class Solution {
    public int[] solution(int[] num_list) {
        int l = num_list.length;
        int[] answer = new int[l + 1];
        
        for (int i=0; i<l; i++) {
            answer[i] = num_list[i];
        }
        
        int a = num_list[l - 1];
        int b = num_list[l - 2];
        if (a > b) {
            answer[l] = a - b;
        } else {
            answer[l] = a * 2;
        }
        
        return answer;
    }
}

File: Programmers/Lv.0/마지막 두 원소/181927.js
function solution(num_list) {
    const answer = [...num_list];
    
    const l = num_list.length;
    const a = num_list[l - 1];
    const b = num_list[l - 2];
    if (a > b) {
        answer.push(a - b);
    } else {
        answer.push(a * 2);
    }
    
    return answer;
}
